packages/inspector-kit/.storybook/preview.js

```diff
@@ -0,0 +1,13 @@
+import 'normalize.css'
+import '@blueprintjs/core/lib/css/blueprint.css'
+import '@blueprintjs/icons/lib/css/blueprint-icons.css'
```

Comment:
Ensures our storybook stories are rendered with the same baseline as our components will be in the studio.